src/components/MockOptionsSelection.vue

```diff
@@ -1,87 +1,91 @@
 <template>
-  <q-dialog persistent position="top" v-model="opened" full-height>
-    <q-card style="width: 550px;">
+  <q-dialog persistent position="top" v-model="opened" full-height style="max-width: 650px;">
+    <q-card style="width: 650px; max-width: 650px;">
       <q-form @submit="submitMockOpt">
-        <q-card-section class="bg-primary shadow-2">
-          <div class="text-h6 text-white" style="text-align: center;">Add mock options to the build</div>
-        </q-card-section>
-        <q-card-section style="width: 550px; padding-left: 25px; padding-right: 25px;">
-          <div>
-            <q-form @submit="onAddMacro" class="q-gutter-md">
-              <div class="row no-wrap group">
-                <q-input v-model="macroInput.name"
-                        :rules="[val => rpmMacroName(val) || 'Invalid macro name']"
-                        lazy-rules
-                        label="Macro Name" clearable
-                        style="width: 242px;">
-                </q-input>
-                <q-input v-model="macroInput.value"
-                        label="Macro Value" clearable
-                        style="margin-left: 20px; width: 242px;"></q-input>
-                <q-item-section side>
-                  <q-btn icon="add" type="submit" v-if="macroInput.name && macroInput.value" flat
-                      small color="primary" round style="left: 15px" size="16px"></q-btn>
-                  </q-item-section>
-              </div>
-            </q-form>
-            <q-list v-if="Object.keys(mock_options.definitions).length" dense highlight no-border>
-              <q-item v-for="macro in mock_options.definitions" :key="macro.name" dense>
-                <q-item-section style="font-size: 12pt; letter-spacing: 1pt;">
-                  --define '{{ macro.name }} {{ macro.value }}'
-                </q-item-section>
-                <q-item-section side style="margin-left: -10px;">
-                  <q-btn @click="onDeleteMacro(macro.name)"
-                        color="negative" flat round small size="16px" icon="delete"/>
-                  </q-item-section>
-                </q-item>
-              </q-list>
-          </div>
+          <q-card-section class="bg-primary shadow-2">
+            <div class="text-h6 text-white" style="text-align: center;">Add mock options to the build</div>
+          </q-card-section>
+          <q-card-section style="width: 650px; padding-left: 25px;">
+            <q-scroll-area style="height: 500px; width: 620px; max-width: 620px;">
+            <div>
+              <q-form @submit="onAddMacro" class="q-gutter-md">
+                <div class="row no-wrap group">
+                  <q-input v-model="macroInput.name"
+                          :rules="[val => rpmMacroName(val) || 'Invalid macro name']"
+                          lazy-rules
+                          label="Macro Name" clearable
+                          style="width: 262px;">
```

Comment:
Why not 260 e.g.?

Reply:
Form width ~540px
15px - button size
540 - 15 = 525px
525 / 2 = 262.5